proto/gen/rill/runtime/v1/runtime.swagger.yaml

```diff
@@ -4340,6 +4340,8 @@ definitions:
         $ref: '#/definitions/v1ExploreSortType'
       exploreExpandedDimension:
         type: string
+      exploreLeaderboardMeasureCount:
```

Comment:
Was this changed by hand? I dont see the field in the proto.

Could you update `resources.proto` and add an entry there? Run `make proto.generate` after to generate these files.

Reply:
This was changed by `runtime.swagger.yaml` and I ran `npm run generate:runtime-client -w web-common`

Reply:
Added